CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # postgres-vacuum-monitor
 
+## v0.17.0
+- Increased default `monitor_max_run_time_seconds` to 60 seconds.
```

Comment:
I wanted to make sure there is enough time for the 5 queries if they all hit the 10 second timeout.